src/IniFile.cpp

```diff
@@ -49,6 +49,9 @@ cIniFile::cIniFile(void) :
 
 bool cIniFile::ReadFile(const AString & a_FileName, bool a_AllowExampleRedirect)
 {
+
+	m_Filename = a_FileName;
+
```

Comment:
And why do you need the entire repository to `Flush`, can't you `Write` the individual underlying INI files instead?